boards/u-blox/ubx_evk_iris_w1/Kconfig.ubx_evk_iris_w1

```diff
@@ -0,0 +1,11 @@
+# EVK-IRIS-W1 board configuration
+# Copyright (c) 2025 u-blox AG
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_UBX_EVK_IRIS_W1
+	select SOC_PART_NUMBER_RW612ETA2I
+
+config FLASH_MACRONIX
+	bool "Macronix Flash support"
+config FLASH_FIDELEX
+	bool "Fidelex Flash support"
```

Comment:
this should not even be a Kconfig, use board variants if both are on the board or board revision if there's 2 different board hardware revisions with the different devices